tests/jvm/src/test/resources/semanticdb.expect

```diff
@@ -29,7 +28,6 @@ Names:
 
 Messages:
 [41..47): [warning] Unused import
-[72..107): [warning] class Stack in package mutable is deprecated (since 2.12.0): Stack is an inelegant and potentially poorly-performing wrapper around List. Use a List assigned to a var instead.
```

Comment:
Why is this message gone?

Reply:
I didn't manage to investigate it. I haven't been able to make use of deprecation messages yet so I was willing to give it up